### PR TITLE
Bumps: wagmi + viem + react query

### PR DESCRIPTION
**Pull request by estebanmino**
Fixes BX-####
Figma link (if any):

### CODE REVIEW

File: src/core/resources/assets/userTestnetNativeAsset.ts [hunk 1]
```diff
@@ -118,16 +119,15 @@ export function useUserTestnetNativeAsset(
     UserTestnetNativeAssetQueryKey
   > = {},
 ) {
-  return useQuery(
-    userTestnetNativeAssetQueryKey({
+  console.log('======= useUserTestnetNativeAsset');
```
Comment (magiziz) [inline src/core/resources/assets/userTestnetNativeAsset.ts:122]: Is this for testing ?

File: src/core/wagmi/createWagmiClient.ts [hunk 2]
```diff
@@ -1,105 +1,127 @@
...
-import { queryClient } from '../react-query';
-import { SUPPORTED_CHAINS, getDefaultRPC } from '../references';
-import { LocalStorage } from '../storage';
-import { ChainId, chainHardhat, chainHardhatOptimism } from '../types/chains';
-import { findRainbowChainForChainId } from '../utils/chains';
+// // import { proxyRpcEndpoint } from '../providers';
+// // import { queryClient } from '../react-query';
+// import { SUPPORTED_CHAINS, getDefaultRPC } from '../references';
+// import { ChainId, chainHardhat, chainHardhatOptimism } from '../types/chains';
+// import { findRainbowChainForChainId } from '../utils/chains';
```
Comment (magiziz) [inline src/core/wagmi/createWagmiClient.ts:8]: Same here. Can we delete this or keep it ?

**Review by magiziz — APPROVED** [review]
Just had some few questions on unused code, but apart from that it looks great 👍

File: src/core/wagmi/createTestWagmiClient.ts [hunk 3]
```diff
@@ -1,70 +1,63 @@
...
-} from 'wagmi';
-import { jsonRpcProvider } from 'wagmi/providers/jsonRpc';
+// import { createAsyncStoragePersister } from '@tanstack/query-async-storage-persister';
+// import {
+//   arbitrum,
+//   avalanche,
+//   base,
+//   bsc,
+//   mainnet,
+//   optimism,
```
[the remark this answers is not in the document]
Comment (estebanmino) [reply src/core/wagmi/createTestWagmiClient.ts:8]: uh yes

File: src/core/wagmi/createWagmiClient.ts (hunk 2 above)
Comment (estebanmino) [reply src/core/wagmi/createWagmiClient.ts:8]: removed the code in your comments 👌  thanks

**Review by BrodyHughes — APPROVED** [review]
LGTM. Wild work Esteban. 

Things tested:
- import flow
- import HW flow
- send and swap flows
- HW send and swap flows
- Went through most settings
- Reset rainbow
- Dapp interactions like signing / sign typed data

If any flows / parts of the bx are concerned, hmu happy to test more if need be.

**Pull request merged**